src/views/InstancesView.js

```diff
@@ -1,6 +1,9 @@
-import React from 'react';
+import React, { memo } from 'react';
```

Comment:
Michal, I've never used memo, but my understanding is that it can only be applied to a functional component, which InstancesView is not. Am I wrong?

Reply:
@Baroquem it should also work with classes: https://github.com/facebook/react/issues/13937

Reply:
Ok then! I like how they basically say yes, it works, but we don't want to document it in case people start using it.

Reply:
And what if we simply convert InstancesView into a functional component for consistency?

Reply:
I like that @SergiyVSergiyenko! I will try to refactor a bit tomorrow.